### ISSUE TITLE
DetailsList column header can't show aria label correctly after calling onRenderColumnHeaderTooltip

### ISSUE THREAD

**Issue by wemmaling** [microsoft/fluentui#21152]
### Environment Information

- **Package version(s)**: version demo used.
- **Browser and OS versions**: Chrome and Edge

### Describe the issue:

Before using the `onRenderColumnHeaderTooltip`, the ariaLabel in `IColumn` can render a a11y label in dom tree. After using it, the label disappeared.

Before(test in [demo page](https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/shimmer)):
![image](https://user-images.githubusercontent.com/16534682/148016557-c4cc91ba-f8d2-4d10-a1b8-44c283f96c32.png)

Afer using `onRenderColumnHeaderTooltip`:

The aria label prop doesn't working, same with `sortAscendingAriaLabel` and `sortDescendingAriaLabel`

https://codepen.io/wemmaling/pen/WNZJdKw

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/wemmaling/pen/WNZJdKw

#### Actual behavior:

No aria label information rendred.

#### Expected behavior:

label with customized aria label exisited.

**Comment by wemmaling** [comment]
Hi @spmonahan, is there any update for this one?

**Comment by gouttierre** [comment]
@wemmaling - Thanks for filing this issue with us. To set expectations, the developers will review this issue once capacity allows as we’re currently focused on larger coherence work items.

@TristanWatanabe - Would you be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by TristanWatanabe** [comment]
@wemmaling The `ariaLabel` prop seems to be working as expected (see screenshot below). `aria-label` is not meant to be rendered on the DOM as a separate label element since it's supposed to be used as a text alternative to help screen reader users. I'm unsure as to how your example with a label rendering on the DOM as text even happened as I was unable to repro that. If you could provide an isolated example where a label ends up being rendered after passing an `ariaLabel`, I think that might be the real issue here.


![image](https://user-images.githubusercontent.com/8649804/154746687-28c2a886-d85e-4d17-99d8-1fc7b0003757.png)
